pom.xml

```diff
@@ -1,6 +1,6 @@
 <project>
     <modelVersion>4.0.0</modelVersion>
-    <groupId>be.ugent.rml</groupId>
+    <groupId>nl.ocs.be.ugent.rml</groupId>
```

Comment:
Let's keep the groupID as it is ;)

Reply:
LOL, oops.  Seems the internal name we used during development accidentally dot into the PR :)